### PR TITLE
Updated README and added levels

### PR DESCRIPTION
**Pull request by MacroMan**
Updated README and added levels:
- Earlytown
- GrandHospital
- HeavenTown
- Old Hillside
- Tetris

The below doesn't really apply here as this is mass adding those that agreed to have them hosted here on Discord

- [x] *I confirm that I have read and applied my submission according to the [Contributing Guidelines](https://github.com/CorsixTH/CommunityContent/blob/main/CONTRIBUTING.md)*

# Community Content Contribution (choose one)
- [x] I am submitting a standalone level
- [ ] I am submitting a map only
- [ ] I am submitting a campaign

**Names of levels/maps:**
- Earlytown
- GrandHospital
- HeavenTown
- Old Hillside
- Tetris

## Campaign Data
- [x] Not applicable
- [ ] I have included a `.campaign` file with this submission

## Game Data
**Please note: All level files used MUST point to a map file to be accepted**

**Level Name:**
The `.level` file is:
- [ ] Not applicable, map only
- [ ] From CorsixTH (specify level file name)
- [x] From a Community Content submission (specify level file name and original author)
- [ ] My own creation and is included in this submission

*Level Declarations (as guided above):*

The `.map` file is:
- [ ] From CorsixTH (specify level file name)
- [x] From a Community Content submission (specify level file and original author)
- [ ] My own creation and is included in this submission

*Map Declarations (as guided above):*

### Your content/files
**Note:** Your authorship declarations must also contain the required reference to the Creative Commons Attribution 4.0 International License (CC BY 4.0)
- [ ] *I confirm that I am the author of this work.*
- [x] All files have been taken from the #community-content discord channel, with permission of the authors
- [x] *I confirm to have put appropriate authorship on all submitted files (including
modified works).*
- [x] *I confirm to have included a `LICENSE.txt` file with my submission.*
### Derivative/redistributed content (complete if applicable)

- [x] *Where portions of this work have been created by other authors (include works
I have modified) I have appropriately declared their authorship and license in the
relevant files.*
- [x] *I confirm that any works from other authors allow for re-use and modification.*
### Agreement
- [x] *I agree for the works to be licensed under Creative Commons Attribution 4.0 International - https://creativecommons.org/licenses/by/4.0/*
- [x] *I confirm that my work is free from offensive (or otherwise sensitive) content.*
- [x] *I understand that CorsixTH will not be responsible for enforcement of the CC BY 4.0 license
except for submissions of falsified authorship (in which case, the offending works
will be removed).*
- [x] *I confirm that I am happy for this work to be uploaded and distributed by the
CorsixTH Community Content repository.*

### CODE REVIEW

File: Levels/GrandHospital/grandhospital.level [hunk 1]
```diff
@@ -0,0 +1,193 @@
+GRAND HOSPITAL is freely distributed under the following terms:
+Copyright (C) 2010 Edvin "Lego3" Linge, 2020 lloyd-lejack, 2024 Matroftt
```
Comment (lewri) [inline Levels/GrandHospital/grandhospital.level]: I don't believe Lloyd is part of this file. Matroftt's year is 2023 (also adjust the license.txt)

File: Levels/Earlytown/Earlytown.level [hunk 2]
```diff
@@ -0,0 +1,263 @@
+EARLYTOWN is freely distributed under the following terms:
+Copyright (C) 2010 Edvin "Lego3" Linge, 2020 lloyd-lejack, 2024 Matroftt
```
Comment (lewri) [inline Levels/Earlytown/Earlytown.level]: I don't believe Lloyd is part of this file. Matroftt's year is 2023 (also adjust the license.txt)

File: Levels/Tetris/tetris-sandbox.level [hunk 3]
```diff
@@ -0,0 +1,212 @@
+TETRIS is freely distributed under the following terms:
+Copyright (C) 2010 Edvin "Lego3" Linge, 2020 lloyd-lejack, 2024 Matroftt
```
Comment (lewri) [inline Levels/Tetris/tetris-sandbox.level]: I don't believe Lloyd is part of this file. Matroftt's year is 2023 (also adjust the license.txt)

File: Levels/Tetris/tetris-scenario.level [hunk 4]
```diff
@@ -0,0 +1,220 @@
+TETRIS is freely distributed under the following terms:
+Copyright (C) 2010 Edvin "Lego3" Linge, 2020 lloyd-lejack, 2024 Matroftt
```
Comment (lewri) [inline Levels/Tetris/tetris-scenario.level]: I don't believe Lloyd is part of this file. Matroftt's year is 2023 (also adjust the license.txt)

File: README.md [hunk 5]
```diff
@@ -1,4 +1,44 @@
-# CorsixTH Community Content
-Fan levels and campaigns
+# CorsixTH custom campaigns, levels and maps
 
-<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```
Comment (lewri) [inline README.md:4]: Could you restore the exact code for this line. Changing to "All works" is fine. A copyright statement should not float all the way to the bottom of the file.

**Review by lewri — COMMENTED** [review]
If you can fill out the form in your original post too that would assist.
I'm aware that this is a bulk upload, comments can be added to areas as necessary; as well as a line saying permission was given by the authors.

**Comment by MacroMan** [comment]
I've made the requested changes and updated the form with relevant info

File: Levels/Hillside + District - V0.3.zip
Comment (lewri) [inline Levels/Hillside + District - V0.3.zip:1]: Remove this file. I will then ask Cyco to request merge

File: README.md [hunk 6]
```diff
@@ -1,4 +1,44 @@
...
+
+### Levels
+
+- Download the `.level` and `.map` files directly and save into the `Levels` directory
+
+### Finding the CorsixTH user directory
+
+| Operating System          | Location (typical)                                             |
+|---------------------------|----------------------------------------------------------------|
+| Linux / macOS             | `${XDG_CONFIG_HOME}/CorsixTH/` or `~/.config/CorsixTH/`        |
```
Comment (Alberth289346) [inline README.md]:
```suggestion
| Linux / macOS             | `${XDG_CONFIG_HOME}/CorsixTH/`, usually `~/.config/CorsixTH/`        |
```

The `XDG_CONFIG_HOME` usually has the value `~/.config`.

Comment (lewri) [reply README.md]: The finding user directory table should likely also be mirrored into a wiki page (heck, even https://github.com/CorsixTH/CorsixTH wiki needs it)

**Comment by lewri** [comment]
Amendments pushed, ready to merge @TheCycoONE

**Pull request merged**